### DISCUSSION TITLE
Is there any plan for Image in Table?

### DISCUSSION THREAD

**Discussion by korjwl1** [slint-ui/slint#4661] [Q&A]
As far as I know, slint only supports text in the table.

<img width="495" alt="image" src="https://github.com/slint-ui/slint/assets/72062804/46eb4c87-895b-4776-8b82-bf7d4d823c25">

However, I want to make a table having image as its content like the image above.
(Above example is made with egui)

Is there any plan to support image as the table content?

**Comment by ogoffart** [comment]
Having a more generic table is something we should try to have. In the mean time, the table need to be done manually https://slint.dev/releases/1.4.1/docs/slint/src/recipes/recipes#custom-table-view
